modules/machine-health-checks-baremetal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * machine_management/deploying-machine-health-checks.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-health-checks-baremetal_{context}"]
+= MachineHealthChecks on Bare Metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
+Usually bare metal reprovisioning is a lengthy process, in which the cluster 
+is lacking compute power, and some applications are down.
+To change the default remediation process from machine deletion to host power-cycle,
+the MachineHealthCheck resource should be annotated with 
+`machine.openshift.io/remediation-strategy: external-baremetal`
```

Comment:
You also need to add this parameter to modules/machine-health-checks-resource.adoc and add an annotation that explains what it is, like:

```
<37> For bare metal clusters, you must add this annotation. Do not add this parameter to any other cluster type.
```

Reply:
I'm concerned that some users will copy the yaml as is and use it, even on non-baremetal cluster which can end up with remediation not working.
WDYT from your experience in other areas?

Reply:
Is there a value for this annotation that's valid for non-baremetal clusters? If so, I'd put that in the YAML and explain the available options, including this one, in the annotation. I expect that people are going to mess this up otherwise because the information is fairly far from where you use it.

Reply:
> Is there a value for this annotation that's valid for non-baremetal clusters?

Unfortunately not. This annotation is only for bare metal with only one valid value.
I'll be happy to do whatever you think is best for the users

Reply:
Then add a callout to the stanza that would include that annotation and provide the entire annotation in the callout text.

Reply:
@kalexand-rh makes sense.
I hope I got it right.